VaticanBAV/et/BAVet15.xml

```diff
@@ -924,7 +924,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <title>Readings and prayers for Sunday</title>
                         <msItem xml:id="p1_i12.1">
                            <locus from="118b" facs="0121"/>
-                           <title>ጸሎት፡ በሰንበተ፡ ክርስቲያን፡</title>
+                           <title ref="LIT4261Prayer"/>
```

Comment:
since the title is provided in the MS and in the catalogue, please keep it, do not delete
`<title ref="LIT4261Prayer">ጸሎት፡ በሰንበተ፡ ክርስቲያን፡</title>`